plugins/color_panel/macos/FLEColorPanelPlugin.mm

```diff
@@ -47,9 +52,10 @@ - (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
 /**
  * Configures the shared instance of NSColorPanel and makes it the frontmost & key window.
  */
-- (void)showColorPanel {
+- (void)showColorPanel:(BOOL)showAlpha {
```

Comment:
ObjC method names should describe the parameter with the part immediately before the :, so this should be `showColorPanelWidthAlpha:`